src/python_testing/TC_DRLK_2_9.py

```diff
@@ -0,0 +1,742 @@
+                                "Error when executing GetUserResponse command, userIndex={}".format(
+                                    str(response.userIndex)))
+            asserts.assert_true(response.userName == username,
+                                "Error when executing GetUserResponse command, userName={}".format(
+                                    str(response.userName)))
+            asserts.assert_true(response.userUniqueID == useruniqueid,
+                                "Error when executing GetUserResponse command, userUniqueID={}".format(
+                                    str(response.userUniqueID)))
+            asserts.assert_true(response.credentials == credentiallist,
+                                "Error when executing GetUserResponse command, credentials={}".format(
+                                    str(response.credentials)))
+            asserts.assert_equal(expected_status, Status.Success)
+
+        except InteractionModelError as e:
+            asserts.assert_equal(e.status, expected_status, f"Unexpected error returned: {e}")
+
+    async def get_credentials_status(self, credentialIndex: int, credentialType: drlkcluster.Enums.CredentialTypeEnum, credential_exists):
+
+        try:
+            credentials_struct = drlkcluster.Structs.CredentialStruct(credentialIndex=credentialIndex,
+                                                                      credentialType=credentialType)
+            response = await self.send_single_cmd(endpoint=self.app_cluster_endpoint, timedRequestTimeoutMs=1000,
+                                                  cmd=drlkcluster.Commands.GetCredentialStatus(
+                                                      credential=credentials_struct))
+            asserts.assert_true(type_matches(response, Clusters.DoorLock.Commands.GetCredentialStatusResponse),
+                                "Unexpected return type for GetCredentialStatus")
+            asserts.assert_true(response.credentialExists == credential_exists,
+                                "Error when executing GetCredentialStatus command, credentialExists={}".format(
+                                    str(response.credentialExists)))
+            if (not credential_exists):
+                asserts.assert_true(response.userIndex == NullValue,
+                                    "Error when executing GetCredentialStatus command, userIndex={}".format(
+                                        str(response.userIndex)))
+            return response
+        except InteractionModelError as e:
+            logging.error(e)
+            asserts.assert_equal(e.status, Status.Success, f"Unexpected error returned: {e}")
+
+    async def set_credential_cmd(self, credential_enum: drlkcluster.Enums.CredentialTypeEnum, statuscode, credentialIndex,
+                                 operationType, userIndex, credentialData, userStatus, userType):
+        custom_stautus_code = 149
```

Comment:
Consider using a global constant for this magic number

Reply:
This comment has not been addressed yet. Not sure where to update the global constant.
 Currently it is not present in init_.py Status enum. Maybe fixing init.py would solve the issue and custom_stautus_code = 149 can be removed